### PR TITLE
Add CentOS

### PR DESCRIPTION
**Pull request by variadico**
This adds instructions on how to build a CentOS 7 cntmp. It includes a Kickstart script that can be used to create an OS tarball.

@earlruby @krobertson @vitroth

### CODE REVIEW

File: os/create-centos-cntmp.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# How to create the CentOS cntmp
+
+The steps in this tutorial were done on CentOS 7.2.1511, using Virtualbox
+5.1.4.
```
Comment (krobertson) [inline os/create-centos-cntmp.md]: Should we maybe have a vagrantfile or something to make launching it easier? Or perhaps link to where you got the image.

Comment (variadico) [inline os/create-centos-cntmp.md]: Good idea.

**Comment by variadico** [comment]
Added a `Vagrantfile`. Everything is automated now. Just gotta do a `vagrant up`.

**Comment by krobertson** [comment]
LGTM

**Pull request merged**